client/src/Movies/MovieList.js

```diff
@@ -24,31 +26,16 @@ export default class MovieList extends Component {
     return (
       <div className="movie-list">
         {this.state.movies.map(movie => (
-          <MovieDetails key={movie.id} movie={movie} />
+          <Link to={`/movies/${movie.id}`}><MovieDetails key={movie.id} movie={movie} /></Link>
```

Comment:
I think that they key needs to be put on the first element right after our map
Please let me know if you think this is incorrect